lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -975,24 +975,30 @@ public function getName()
      */
     protected function doModifyLimitQuery($query, $limit, $offset = null)
     {
-        $limit = (int) $limit;
```

Comment:
@deeky666 The confusing part here is how is `$offset = null` different from `$offset = 0` from the SQL semantics standpoint? Should the DBAL produce additional sub-query in the latter case? Right not it doesn't which makes more sense to me.